pages/blog/hackathon-faq.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "AsyncAPI Hackathon FAQ"
+date: 2021-10-19T06:00:00+01:00
+type: Communication
+tags:
+  - Conference
+  - Hackathon
+cover: /img/posts/hackathon-faq/cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+excerpt: Answers to all the questions you ever wanted to ask about the AsyncAPI Hackathon 2021
+featured: true
+---
+
+Last year it was the first time we hosted the AsyncAPI Conference. This year it is time to host our first AsyncAPI Hackathon. Many questions are coming from the community, and this article tries to address them.
+
+### When is the Hackathon?
+
+1st - 31st of October 2021.
+
+You have an entire month to complete your idea. No usual over-weekend hard work but relaxed month instead.
+
+### What is the main theme for the Hackathon?
+
+The goal of the Hackathon is to provide MVP solutions that can help AsyncAPI Community in ways like:
+
+- Ease education and getting started with the project.
+- Bring new AsyncAPI-related solutions that can solve the existing challenges the community is facing with event-driven architectures.
+- Extend existing tools, reuse them in some scenarios, add the extra implementation on top.
+
+### Do I need to register for the event?
+
+No.
+
+Anybody can join the Hackathon. Whenever you want. No pressure in case you can't deliver what you planned.
```

Comment:
```suggestion
No, you do not need to register for the event. 🙂

Anybody can join the Hackathon, whenever they want! No pressure, in case you realize you can't deliver what you had originally planned.
```